config/i18n-tasks.yml

```diff
@@ -94,6 +94,7 @@ ignore_unused:
 - 'errors.not_authorized'
 - 'experiments.*'
 - 'event_types.*'
+- 'headings.create_account_with*' # cannot find with interpolation
```

Comment:
you mean an entirely new view template?

I would vote not to go the route of separate view templates for with SP / without SP.  I think having entirely separate view templates for just a few pieces of different code will *very* likely lead to updating one view and not the other.

Although in general I feel like developers are way more obsessed with getting rid of duplication than they need to be, this is a case where I can easily see it becoming problematic for us since almost all views are going to have these "with sp" and "without sp" snippets...which would mean 2x for each view template in the app.

The existing pattern for handling this includes using conditionals in the views: https://github.com/18F/identity-idp/blob/master/app/views/devise/sessions/new.html.slim#L3

I also find that pattern undesirable. I've come onto projects in past months with tons of conditionals in the views and they can get really messy. Soon enough, views have `elsif` statements and nested conditionals and it becomes impossible to know what is what.

So, I feel like I'd rather use this pattern of interpolation and have `t('key.key.#{helper_method}` all over the place than have 2 files for each view with lots of duplication or have views riddled with conditionals.

Other ideas on how we can tackle this?